libs/sdk-ui-kit/src/Datepicker/Datepicker.tsx

```diff
@@ -93,6 +99,14 @@ function parseDate(str: string, dateFormat: string): Date | undefined {
     }
 }
 
+function convertWeekStart(weekStart: WeekStartType): 0 | 1 | 2 | 3 | 4 | 5 | 6 {
+    if (weekStart === "Monday") {
```

Comment:
I would prefer to change this structure into `switch` statement, return explicitly value for `Monday` and `Sunday,` and throw error in `default` branch.